wiremockmanager/wmm.py

```diff
@@ -23,33 +23,37 @@ def wrapper(**kwargs):
         return func(**kwargs)
     return wrapper
 
+
 @argh.decorators.named('mock')
 @argh.arg('--api', default='', help='API folder name')
 @argh.arg('--version', default='', help='Version folder name')
 @argh.arg('--port', default=0, help='HTTP port to use')
 @argh.arg('--https-port', default=0, help='HTTPS port to use')
+@argh.arg('--https-keystore', default='', help='keystore file containing an SSL certificate')
+@argh.arg('--keystore-password', default='', help='Password to the keystore, if something other than password.')
 @validate_directory
 @initialize_directory
-def mock(api, version, port, https_port):
+def mock(api, version, port, https_port, https_keystore=None, keystore_password=None):
     """
     Start an instance to mock of the specified API and version. This will serve the defined behaviors located in the
     directory 'services/[api]/[version]'.
 
     If 'services/[api]/[version]' does not exist, this command will return with an error.
     """
     try:
-        instance = _mock(api, version, port, https_port)
+        instance = _mock(api, version, port, https_port, https_keystore, keystore_password)
         _print_table([instance])
     except workspace.WorkspaceError as wse:
         _print_message(wse.message)
     except wiremock.WireMockError as wme:
         _print_message("Could not start WireMock instance. Please see log file for more details: {}".format(wme.message))
 
 
-def _mock(api, version, port, https_port):
+def _mock(api, version, port, https_port, https_keystore, keystore_password):
```

Comment:
`https_keystore` and `keystore_password` should default to `None` here as well

Reply:
@dankolbrs right problem, wrong solution.